src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -285,10 +285,25 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
       case (proposals, bakersBlocks, ballotsBlocks) =>
         //this is a nested list, each block with many baker rolls
         val writeBakers = bakersBlocks.traverse {
-          case (block, bakersRolls) => TezosDb.writeVotingRolls(bakersRolls, block)
+          case (block, bakersRolls) =>
+            TezosDb.writeVotingRolls(bakersRolls, block)
+        }
+
+        val updateAccountsHistory = bakersBlocks.traverse {
+          case (block, bakersRolls) =>
+            TezosDb.updateAccountsHistoryWithBakers(bakersRolls, block)
+        }
+
+        val updateAccounts = bakersBlocks.traverse {
+          case (block, bakersRolls) =>
+            TezosDb.updateAccountsWithBakers(bakersRolls, block)
         }
 
-        val combinedVoteWrites = writeBakers.map(_.combineAll.map(_ + proposals.size + ballotsBlocks.size))
+        val combinedVoteWrites = for {
+          bakersWritten <- writeBakers
+          accountsHistoryUpdated <- updateAccountsHistory
+          accountsUpdated <- updateAccounts
+        } yield bakersWritten.combineAll.map(_ + proposals.size + ballotsBlocks.size + accountsHistoryUpdated.size + accountsUpdated.size)
```

Comment:
The final sum of all data is meant to be logged in the call to `logVotingOutcome`, I'm not sure that the logging is still up-to-date with the numbers being given from here